tests/unit/cypher_main_visitor.cpp

```diff
@@ -3707,43 +3724,57 @@ TEST_P(CypherMainVisitorTest, CreateStream) {
 
     const auto topic_names_as_str = utils::Join(topic_names, ",");
 
-    ValidateCreateStreamQuery(
+    ValidateCreateKafkaStreamQuery(
```

Comment:
I would vary to order of the different config values to make sure there is really no ordering between them.

Reply:
How does it look now?